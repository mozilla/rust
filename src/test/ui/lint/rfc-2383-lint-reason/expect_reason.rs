// check-pass

#![feature(lint_reasons)]

#![expect(unused_variables, reason = "<This should fail and display this reason>")]
//~^ WARNING this lint expectation is unfulfilled [unfulfilled_lint_expectation]
//~| NOTE `#[warn(unfulfilled_lint_expectation)]` on by default
//~| NOTE <This should fail and display this reason>

fn main() {}
