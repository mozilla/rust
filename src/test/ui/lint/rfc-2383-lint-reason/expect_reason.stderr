warning: this lint expectation is unfulfilled
  --> $DIR/expect_reason.rs:5:1
   |
LL | #![expect(unused_variables, reason = "<This should fail and display this reason>")]
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: `#[warn(unfulfilled_lint_expectation)]` on by default
   = note: <This should fail and display this reason>

warning: 1 warning emitted

